frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -219,10 +222,25 @@ function FileSearcher:isFileMatch(filename, fullpath, keywords, is_file)
                     end
                 end
             end
+        else
+            self.no_metadata_count = self.no_metadata_count + 1
         end
     end
 end
 
+function FileSearcher:showSearchResultsMessage(no_results)
+    local text = no_results and T(_("No results for '%1'."), self.search_value)
+    if self.no_metadata_count ~= 0 then
+        local txt = T(N_("1 book has not been searched in.", "%1 books has not been searched in.",
+            self.no_metadata_count), self.no_metadata_count) .. "\n" ..
+            _("Please do 'Extract and cache book information' from Plus menu to search in all books metadata.")
```

Comment:
May be:
You can ensure that all book metadata will be inspected by calling 'Extract and cache book information' from the \u{2795} menu.